aws/iam/helpers.py

```diff
@@ -0,0 +1,85 @@
+from dateutil.parser import parse
+
+
+def user_is_inactive(iam_user, grace_period, considered_inactive):
+    """
+    Returns False if any of these are true:
+        - The user was created less than the grace period
```

Comment:
If I'm understanding this correctly this is "before the account creation grace period has elapsed" or "within the account creation first use grace period."